server/hl7/hl7.controller.js

```diff
@@ -186,4 +204,7 @@ function _getMessageByIdOrIndex(value, fileId) {
   });
 }
 
-module.exports = { parseFile, upload, getUserFiles, getMessageByIndex, getMessageByid };
+module.exports = {
+  parseFile, upload, getUserFiles, getMessageByIndex,
+  getMessageByid, getFile
```

Comment:
Since we're breaking this into multiple lines, I would personally just give each export its own line.

Reply:
+1 we actually get the `Object-property-newline` linting error if they are not all on the same line or each on its on line.